### ISSUE TITLE
NetBeans installer won't recognize Azul Zulu JDK

### ISSUE THREAD

**Issue by zzador** [apache/netbeans#6389]
### Apache NetBeans version

Apache NetBeans 18

### What happened

Tried the netbeans installer as the binary package won't decompress because the zip file is broken. Once started it said I wouldn't have the necessary JDK installed but I have installed Zulu Java 8 JDK, Zulu Java 11 JDK and Zulu Java 17 JDK.

PLEASE remove the JDK detection from the installer. JDK detection is just unnecessary harassment cause Oracle isn't the only player anymore.

### How to reproduce

Install a Zulu JDK (or any other non Oracle JDK) as the only JDK on a fresh Windows 10 and try the NetBeans installer.

### Did this work correctly in an earlier version?

No / Don't know

### Operating System

Windows 10

### JDK

17

### Apache NetBeans packaging

Apache NetBeans provided installer

### Are you willing to submit a pull request?

No

**Comment by zzador** [comment]
Nope. JAVA_HOME is set to a valid Azul ZULU Java 17 JDK installation path. All other tools that depend on the JAVA_HOME variable are working fine. Still can't install NetBeans 18 IDE cause of Missing-JDK-Error.

**Comment by emaphis** [comment]
The NetBeans18 installation program worked fine for me on Windows 10 with Zulu JDK.
I've been testing NetBeans19 so I've been installing and un-installing NetBeans on Windows anyway, so I tested the NetBeans18's  "exe" installation program on Windows 10 just for giggles.
I have Zulu 8, 11, and 17 installed in my "C:\Program Files\Java\" directory. I have Zulu 17 as the default so it's set up in my PATH and JAVA_HOME environmental variables on boot up.
The installation program found Zulu 17 just fine and also found the Zulu 8 and 11 installations   The installation JDK text box drop-down showed all of the JDKs installed in the "C:\Program Files\Java\".  
I uninstalled NetBeans18 and then installed the test build of NetBeans19 with the same result.
